src/natcap/invest/wave_energy.py

```diff
@@ -287,26 +287,20 @@
         },
         "machine_param_path": {
             "type": "csv",
-            "rows": {
-                "capmax": {
-                    "about": gettext("Maximum capacity for device."),
-                    "type": "number",
-                    "units": u.kilowatt
-                },
-                "hsmax": {
+            "columns": {
```

Comment:
I switched this to columns so that we don't have to validate the individual rows, as discussed on the last team call.

Reply:
Yep, makes sense!  Could you add an inline comment about this so we are sure to remember?